client_app/src/components/shelter/RequestForHelp.js

```diff
@@ -5,6 +5,8 @@ import dayjs from "dayjs";
 import "react-big-calendar/lib/css/react-big-calendar.css";
 import { Modal, Button, Form } from "react-bootstrap";
 
+const SERVER = "http://localhost:5000";
```

Comment:
We have the SERVER defined in ```client_app/src/config.js``` file. You should use that definition instead of re-defining it here.